src/main/resources/order-api.yml

```diff
@@ -0,0 +1,208 @@
+      responses:
+        "200":
+          description: Result of the operation
+          schema:
+            type: long
+
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    /order/{order_id}:
+      get:
+        summary: Fetch an order
+        description: The endpoint returns information about an order with a given id.
+        The response includes the name, date and status of the order.
+        parameters:
+        - name: order_id
+          in: url
+          description: Order id
+          required: true
+          type: string
+        security:
+        - token: []
+        tags:
+        - order details
+        responses:
+          "200":
+            description:  Order fields: name, date and status
+            schema:
+              type: json
+              order:
+                $ref: '#/definitions/Order'
+          "403":
+            description: Unauthorized for the current operation
+            schema:
+              $ref: '#/definitions/Error'
+          "404":
+            description: Order not found
+            schema:
+              $ref: '#/definitions/Error'
+          default:
+            description: Unexpected error
+            schema:
+              $ref: '#/definitions/Error'
+
+      /order/{order_id}/status:
+      put:
+        summary: Updates current order status
+        description: The endpoint updates current status of the order with given id.
+        The response includes the result of the operation.
+        parameters:
+        - name: id
+          in: url
+          description: Order id
+          required: true
+          type: string
+        - name: status
+          in: url
+          description: Order status
+          type: string
+        security:
+        - token: []
+        tags:
+        - order status
+        requestBody:
+          content:
+            application/text:     # Media type
+              examples: pending
+        responses:
+          "200":
+            description:  Order status
+            schema:
+              type: string
+          "403":
+            description: Unauthorized for the current operation
+            schema:
+              $ref: '#/definitions/Error'
+          "404":
+            description: Order not found
+            schema:
+              $ref: '#/definitions/Error'
+          "409":
+            description: Conflict: the order has been already taken by another business
+            schema:
+              $ref: '#/definitions/Error'
+          default:
+            description: Unexpected error
+            schema:
+              $ref: '#/definitions/Error'
+definitions:
+  Order:
+    properties:
+      order_id:
+        type: string
+        description: Unique identifier representing a specific order.
+      business_id:
+        type: string
+        description: Unique identifier representing a business, which accepted the order.
+      status:
```

Comment:
It may make sense to mention that it may be null before any business accepts it.

Reply:
And we should explicitly list all possible statuses for the status. There is an enumeration type for this in yaml.

Reply:
thanks for the hint, changed it